requirements/dev.txt

```diff
@@ -12,7 +12,7 @@ aniso8601==9.0.1
     # via
     #   -r requirements/quality.txt
     #   tincan
-asgiref==3.4.1
+asgiref==3.5.0
```

Comment:
- Dropped support for `Python 3.6`.  No further major change: https://github.com/django/asgiref/blob/main/CHANGELOG.txt